README.md

```diff
@@ -52,6 +52,13 @@ ignoreSigRegexps:
 ignorePackageGlobs:
 - encoding/*
 - github.com/pkg/*
+
+# An array of strings which specify regular expressions of interface names.
```

Comment:
Maybe we could make this a little more succinct. I liked the description on the config struct more, maybe we could use that here.

```
// IgnoreInterfaceRegexps defines a list of regular expressions which, if matched
// to a underlying interface name, will ignore unwrapped errors returned from a
// function whose call is defined on the given interface.
```